util/src/main/java/tc/oc/pgm/util/text/types/PlayerComponent.java

```diff
@@ -92,7 +92,8 @@ static Component of(
 
   // What an offline or vanished username renders as
   static TextComponent.Builder formatOffline(String name, boolean plain) {
-    TextComponent.Builder component = TextComponent.builder().append(name);
+    TextComponent.Builder component =
+        TextComponent.builder().append(TranslatableComponent.of(name));
```

Comment:
What's the reason behind wrapping the string in a translatable component? 🤔

Reply:
Right, I wanted to use a translatable string key, but I realize I should require a `Component` instead of a `String` there in the first place. (Instead of hardcoded "Unknown" as it was previously I wanted "misc.unknown")